packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.ts

```diff
@@ -59,40 +55,22 @@ const OverrideResolverLoader = function (this: LoaderContext<any>) {
     const applicationExtensions = compiler?.custom?.extensions || []
 
     // Get the master list of all possible candidate paths based on your current extension configuration.
-    let paths = buildCandidatePaths(projectRelPath, {
+    const paths = buildCandidatePaths(projectRelPath, packageName, {
```

Comment:
The only concern here is with Windows. Since the `packageName` will be `@salesforce/extension-sample`, I'm worried that the forward slash could cause issues on Windows. So we may need to use Node's `path` module to avoid any issues here.

Reply:
Ok.. so I've added some code to address your concern, which is valid btw. But I can't test this on windows (although I think it's going to work) until the windows fix PR is merged.